openlibrary/plugins/openlibrary/js/add-book.js

```diff
@@ -36,6 +37,9 @@ export function initAddBookImport () {
     $('#addbook').on('submit', parseAndValidateId);
     $('#id_value').on('input', clearErrors);
     $('#id_name').on('change', clearErrors);
+    $('#id_value').on('paste', debounce(() => {
+        checkIsbn();
+    }, 500, false));
```

Comment:
```suggestion
```
Let's remove this handler altogether.  There is already validation for IDs, and the ISBN lookup should be included in those checks.

Also, some folks will be keying in the ISBNs (rather than pasting them).